lib/matplotlib/axes/_axes.py

```diff
@@ -2970,9 +2975,19 @@ def pie(self, x, explode=None, labels=None, colors=None,
             raise ValueError("Wedge sizes 'x' must be non negative values")
 
         sx = x.sum()
-        if sx > 1:
-            x = x / sx
 
+        if normalize is None:
+            if sx < 1:
+                cbook.warn_deprecated(
+                    "3.1", message="normalize=None gives the current behavior "
+                    "but this behavior is deprecated "
+                    "since %(since)s. After the deprecation period "
+                    "the default value will be normalize=True. "
+                    "To reproduce the current behavior pass normalize=False ")
+                    
+        if normalize or sx > 1:
```

Comment:
To align the code more with the future logic and confine the temporary `None` handling in one block, I would put

~~~
if normalize is None:
    if sx < 1:
        ...
    else:
        normalize = True
~~~

and here just
```suggestion
        if normalize:
```
This also makes it easier to remove the `None` handling after the deprecation.